.github/workflows/performancetest.yml

```diff
@@ -0,0 +1,40 @@
+name: Performance-Test
+
+on:  
+  push:
+    branches:
+      - development
+
+jobs:
+  slsart-perf:
+    name: slsart Performance Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    env:
```

Comment:
Why did you decide to put the env in the job and not at the top level? I don't necessarily have an opinion on where it should go, but I'm curious as to the reasoning?

Reply:
Thought making them job specific would be the safest, not really sure how much it matters though.